src/main/java/taxi/model/Driver.java

```diff
@@ -27,6 +35,22 @@ public String getName() {
         return name;
     }
 
+    public String getLogin() {
+        return login;
+    }
+
+    public void setLogin(String login) {
+        this.login = login;
+    }
+
+    public String getPassword() {
+        return password;
+    }
+
+    public void setPassword(String password) {
+        this.password = password;
+    }
+
```

Comment:
override methods equals() and hashcode() with new parameters - login and password